crates/rolldown_common/src/ecmascript/node_builtin_modules.rs

```diff
@@ -4,10 +4,9 @@ use oxc_resolver::NODEJS_BUILTINS;
 const NODEJS_PREFIXED_BUILTINS: &[&str] = &[
   // https://nodejs.org/api/modules.html#built-in-modules-with-mandatory-node-prefix
   "node:sea",
+  "node:sqlite",
   "node:test",
   "node:test/reporters",
-  // https://nodejs.org/api/sqlite.html
```

Comment:
removed this comment as `node:sqlite` is now included in the list of https://nodejs.org/api/modules.html#built-in-modules-with-mandatory-node-prefix